request_query_count/query_count.py

```diff
@@ -0,0 +1,267 @@
+import json
+import re
+import sys
+from sys import maxsize, stderr
+
+ANY = '<any>'
+
+
+class QueryCountExclusion(object):
```

Comment:
I think we should add a blank line between class or docstring and the next variable or function definition.